### PR TITLE
feat(models): Adding unit tests for models

### PR DESCRIPTION
**Pull request by marklee22**
**What:** Adding unit tests to movie model.

**Why:** Following guide.

### CODE REVIEW

**Review by sjl2 — APPROVED** [review]
L G T M

File: package.json [hunk 1]
```diff
@@ -34,7 +35,7 @@
     "release:minor": "changelog -m && git add CHANGELOG.md && git commit -m 'updated CHANGELOG.md' && npm version minor && git push origin && git push origin --tags",
     "release:major": "changelog -M && git add CHANGELOG.md && git commit -m 'updated CHANGELOG.md' && npm version major && git push origin && git push origin --tags",
     "start": "nodemon --ignore test/",
-    "test": "NODE_ENV=test istanbul cover _mocha -- test --require test/setup.js --recursive --timeout 30000",
+    "test": "yarn lint && NODE_ENV=test istanbul cover _mocha -- test --require test/setup.js --recursive --timeout 30000",
```
Comment (kienan) [inline package.json]: Can you take the `yarn lint` out of here? I think it's best to have your scripts separate and then if you want to run tests and lint locally just run `yarn test && yarn lint`?

**Review by kienan — APPROVED** [review]
LGTM with a minor comment!

File: package.json (hunk 1 above)
Comment (marklee22) [reply package.json]: Good point. I'll update my workflow to do that instead.

**Comment by kienan** [comment]
@marklee22 what branch did this get merged into? I didn't pick up on it before but all PRs should be merged into the master branch unless you're working on a feature that requires multiple PRs

**Comment by marklee22** [comment]
I'm looking into that now as I realized my diffs aren't rendering properly on github. Looks like it's getting merged into my other feature branch.

I'll fix it.

**Comment by kienan** [comment]
@marklee22 let me know if you need any help 👌

**Pull request merged**